.github/workflows/flowzone.yml

```diff
@@ -1334,6 +1334,38 @@ jobs:
           json: ${{ toJSON(inputs) }}
           secrets: ${{ toJSON(secrets) }}
 
+  custom_always:
+    name: Always custom
+    runs-on: ${{ fromJSON(inputs.runs_on) }}
+    timeout-minutes: ${{ fromJSON(inputs.jobs_timeout_minutes) }}
+    needs:
+      - event_types
+      - project_types
+      - versioned_source
+      - custom_test
+      - custom_publish
+      - custom_finalize
+      - custom_clean
+    if: always()
+
+    steps:
+      - name: Download source artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: source-${{ github.event.pull_request.head.sha || github.event.head_commit.id }}
+          path: /tmp
+
+      - name: Extract source artifact
+        working-directory: .
+        shell: bash
+        run: tar -xvf /tmp/source.tgz
+
+      - uses: ./.github/actions/always
```

Comment:
We need to add checks that this file exists, like we've done for the other custom actions.

When running other repos the example actions do not exist.